File: CrossfitDiary/CoreApp/CrossfitDiaryCore.Web/ClientApp/models/viewModels/PersonMaximumViewModel.ts
import Deserializable = General.Deserializable;
export class PersonMaximumViewModel implements Deserializable {
  exerciseId: number;
  exerciseTitle: string;
  maximumWeight?: number;
  maximumAlternativeWeight?: number;
  calculatedMaximumWeight: number;

  deserialize(input: any): this {
    Object.assign(this, input);
    return this;
  }
}

File: CrossfitDiary/CoreApp/CrossfitDiaryCore.Web/ViewModels/PersonMaximumViewModel.cs
using Newtonsoft.Json;

namespace CrossfitDiaryCore.Web.ViewModels
{
    public class PersonMaximumViewModel
    {

        [JsonProperty("exerciseId")]
        public int ExerciseId { get; set; }

        [JsonProperty("exerciseTitle")]
        public string ExerciseTitle { get; set; }

        [JsonProperty("maximumWeight")]
        public double? MaximumWeight { get; set; }

        [JsonProperty("maximumAlternativeWeight")]
        public double? MaximumAlternativeWeight { get; set; }

        [JsonProperty("calculatedMaximumWeight")]
        public double? CalculatedMaximumWeight { get; set; }
    }
}